### DISCUSSION TITLE
Proposal: Renaming the Astro docs to the Astronomicon

### DISCUSSION THREAD

**Discussion by BenDMyers** [withastro/roadmap#279] [Proposal]
I humbly propose naming Astro's docs the Astronomicon. This would reinforce the documentation's position as a trove of arcane knowledge that unleashes terrible power on whomever would wield it. One could say too much power, even. This could be seen as an opportunity to differentiate Astro from other web frameworks. Could one envision an Eleventynomicon? It wouldn't be quite right. Forbidden, even.

Please humbly consider this proposal, and strive to appease the Great Old One who watches over wayfarers in the heavens.

**Comment by nateemerson** [comment]
> Forbidden, even.

+1. Unique value proposition that only Astro can fulfill.

**Comment by ajcwebdev** [comment]
Important context, the suffix `-nomicon` derives from the "background of evil [verisimilitude](https://en.wikipedia.org/wiki/Verisimilitude)," described in the [Necronomicon](https://en.wikipedia.org/wiki/Necronomicon). Verisimilitude (or truthlikeness) is the notion that some [propositions](https://en.wikipedia.org/wiki/Proposition) are closer to being [true](https://en.wikipedia.org/wiki/Truth) than other propositions. I believe the Astro docs are striving to be true versus not being true, but do it in a non-evil way. Thus the comparison is apt.

**Comment by jasikpark** [comment]
Similar to the [Rustnomicon](https://doc.rust-lang.org/nomicon/) :)

**Comment by thewildmage** [comment]
I’m not sure this is a good idea given the way the Necronomicon is presented in Lovecraft’s work. For one example, I think the [“Criticism” section of “History of the Necronomicon” on Wikipedia](https://en.wikipedia.org/wiki/History_of_the_Necronomicon#Criticism) makes some points that should be considered before renaming a thing in homage to the Necronomicon.

If the name change does go through, maybe it would be good to then also address the potentially problematic association created? That way it’s clear what about the Necronomicon the name “Astronomicon” is meant to evoke.

**Reply by BenDMyers** [reply]
This is a great response, thank you - I knew Lovecraft was a raging racist, antisemite, and all around terrible person, but I wasn't aware that the Necronomicon involved anti-Middle Eastern xenophobia. Thank you for informing me!

I suppose I should clarify that I intended this proposal entirely jokingly, and I didn't actually intend for a real name change to come out of this.

**Reply by thewildmage** [reply]
Thank you for hearing me out! I also knew that Lovecraft was terrible and this discussion made me stop and wonder, “Wait, is the Necronomicon itself problematic, too?” So thank you for helping me learn something today as well!

I shared just in case it wasn’t, but I had a feeling this was all meant as a joke. I thought the proposal was well-written, definitely made me chuckle!
